core-java/src/test/java/com/baeldung/junit4vstestng/SortedTests.java

```diff
@@ -0,0 +1,25 @@
+package com.baeldung.junit4vstestng;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class SortedTests {
+    
+    @Test
+    public void a_givenString_whenChangedtoInt_thenTrue(){
+        assertTrue(
```

Comment:
the code is good, but you can keep it a single line. The manual line break is important only inside the article. You can fix it in all tests and I'll merge and suggest to publish the changes, best